lib/srv/db/cloud/watchers/watcher_test.go

```diff
@@ -51,7 +51,7 @@ func TestWatcher(t *testing.T) {
 		AWSMatchers: []services.AWSMatcher{
 			{
 				Types:   []string{services.AWSMatcherRDS},
-				Regions: []string{"us-east-1"},
+				Regions: []string{"us-east-1", "ca-central-1"},
```

Comment:
Can you please make sure these tests cover all scenarios you mentioned in the description?